setup.py

```diff
@@ -5,4 +5,5 @@
 import setuptools
 
 if __name__ == '__main__':
+
```

Comment:
let's remove this change and minimize our diff

Reply:
@yangjunqingqiao please take care of this

Reply:
I'm confused what should happen for this, Flake8 will complain if I revert this change (no ending newline). Should I ignore the specific error or is there a way to untrack the change in the PR?

Reply:
It's just that there should not be a space between the line `if __name__ == '__main__':` and `setuptools.setup`. There should be an ending newline, which is unrelated to this line of diff